### ISSUE TITLE
Strict Concurrency Warnings

### ISSUE THREAD

**Issue by sharpwestOFW** [Alamofire/Alamofire#3887]
* [X ] I've read, understood, and done my best to follow the [*CONTRIBUTING guidelines](https://github.com/Alamofire/Alamofire/blob/master/CONTRIBUTING.md).

I couldn't find any other issues that were opened related to this issue, but I apologize if this isn't the right place to post this.

## What did you do?
Enabled StrictConcurrency in my custom swift package using 
```
.target(
	name: <PackageName>,
	swiftSettings: [.enableExperimentalFeature("StrictConcurrency")]
),
```

## What did you expect to happen?
I expected not to have extra build warnings from Alamofire

## What happened instead?
I received several build warnings related to things not being marked as Sendable. In my package so far, I have "non-sendable type" warnings for:
Session 
RequestInterceptor
DataRequest

## Alamofire Environment

**Alamofire Version:** 5.9.1
**Dependency Manager:** Swift Package Manager
**Xcode Version:** 15.4 AND 16.0 beta 3
**Swift Version:** 5.9
**Platform(s) Running Alamofire:** iOS
**macOS Version Running Xcode:** 14.6

**Comment by jshier** [comment]
Unfortunately Alamofire's concurrency story is complex and Apple's framework support is incomplete. You can retroactively conform the relevant types using `@unchecked Sendable`, as those types are already safe. As the Xcode 16 betas progress I'm fixing the warnings and issues in the `swift6` branch and will ship an update once that stabalizes.

**Comment by sharpwestOFW** [comment]
@jshier Thank you for the fast response, Im looking forward to the future changes!!

**Comment by jshier** [comment]
You can watch PR #3880 for progress here, I should be able to release it soon.

**Comment by jshier** [comment]
This has shipped as part of 5.10.
